gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -341,13 +335,6 @@ public Boolean load(String bucketName) {
         MetricsSink.CLOUD_MONITORING == options.getMetricsSink()
             ? CloudMonitoringMetricsRecorder.create(options.getProjectId(), finalCredentials)
             : new NoOpMetricsRecorder();
-
-    // Create the gRPC stub if necessary;
-    if (options.isGrpcEnabled()) {
```

Comment:
Let use remove the java stroage flag and use gRPC flag to use Veneer client? grpc.enable flag will not have any effect, I guess?

Reply:
I am keeping track of it. As other flag is already shared with customer in preview phase will not change it immediately. Will be updating it in follow up PR.